### ISSUE TITLE
Plants Sprite Refresh Error?

### ISSUE THREAD

**Issue by catmanpop** [ParadiseSS13/Paradise#15401]
TL;DR : Grab plant with 1 hand missing breaks player, making them a plant without holding the plant

**Issue Description**:

Grabbing a Plant with a missing hand will stop the player's sprite from refreshing and leaving them as a plant

**What did you expect to happen**: 

Just a graphical issue leaving the player as a plant 

**What happened instead**:

I grabbed a plant and my sprite bugged out
**Steps to reproduce the problem**:

1. Loose a hand (I lost my right one)
2. Grab a plant
3. Become a plant

**When did the problem start happening**:

Jan 30*
Round ID 25638

**Extra information**:

Cured by grabbing a plant after getting the hand reattached
